### DISCUSSION TITLE
How to reduce brightness and contrast

### DISCUSSION THREAD

**Discussion by DrDawzee** [dlemstra/Magick.NET#1649] [Help] [answered]
Hello to everyone!
I want to decrease the brightness and contrast of an image, but I can't pass negative values to MagickImage.BrightnessContrast (since Percentage doesn't accept negative values).
Am I missing something?
Is there another way to modify brightness and contrast?
This should be trivial, but I can't understand how to handle the issue and couldn't find anything online.

Thanks to anyone who can help.

**Comment by quebecguy** [comment]
With one of the latest update of Magick.NET, some guards has been added in  the Constructor of Percentage to prevent negative value.  Honestly, I don't know why because everything was working fine before setting brightness and contrast with negative percentage.  All I can say is that at least Magict.NET.Core version 13.6.0.0 is still ok, as the guards of negative values are not there.

Just make sure to use nuget package not containing the negative guards.

**Comment by dlemstra** [comment, marked as answer]
These changes were made in #1572 and it looks like I made a mistake that negative percentages were never used. I will publish a new version tomorrow that fixes this again.

cc @Gounlaf

**Reply by Gounlaf** [reply]
Oops :(

Edit: could you add some comments in the class to explain why it can be negative?

**Reply by dlemstra** [reply]
No need to say oops, just wanted to give you an update. This was my mistake. And yes I will add a comment why a negative value is allowed.

**Reply by Gounlaf** [reply]
It's a collaborative mistake 😄

**Reply by DrDawzee** [reply]
Just tested the new release and it works as expected.
Thank you so much!
